src/main/resources/META-INF/nix-idea-ultimate.xml

```diff
@@ -14,4 +12,4 @@
                 implementation="org.nixos.idea.lsp.NixLspServerSupportProvider"/>
 
     </extensions>
-</idea-plugin>
+</idea-plugin>
```

Comment:
nit: While I haven't seen it documented explicitly, I think it is Unix or GNU convention to keep an empty line at the end of all text files. I usually don't call this out when people forget the empty line. It is not important, after all. But since this change does nothing else than removing this line, I thought I should mention it.